yt_dlp/extractor/generic.py

```diff
@@ -2414,6 +2414,7 @@ class GenericIE(InfoExtractor):
                 'channel_url': 'https://rumble.com/c/c-546523',
                 'thumbnail': 'https://sp.rmbl.ws/s8/1/5/f/x/x/5fxxb.OvCc.1-small-Moose-The-Dog-Helps-Girls-D.jpg',
                 'duration': 103,
+                'live_status': 'not_live',
```

Comment:
not strictly related to the PR, but could u move these tests into rumble.py?
It'll become `_WEBPAGE_TESTS = [...]` (and run as `test_Rumble_webpage_all`)